vendor/src/github.com/Graylog2/go-gelf/LICENSE

```diff
@@ -0,0 +1,21 @@
+Copyright 2012 SocialCode
```

Comment:
Has someone confirmed the LICENSE is okay? Not sure what the status of acceptable vendored licenses is?